### DISCUSSION TITLE
Additional Diagnostics data via MQTT

### DISCUSSION THREAD

**Discussion by chunkysteveo** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#228] [Ideas]
Only because:

..."ALL THE DATA"...

Would it be possible to pass in additional device data to the MQTT messages (or another topic) for:

- Wifi RSSI
- WiFi SSID
- IPv4

Just looking at the wealth of data that Tasmota and ESPHome gather - it would be nice to have this to hand (to then do nothing with in Home Assistant!!!!)

![tasmota-sensors](https://user-images.githubusercontent.com/14663002/157687570-24673970-b8a2-4b2f-90e8-01027cd5b2b9.jpg)

Steve

**Comment by visualapproach** [comment]
Ok I hear you 😄 
The answer is yes, and as a matter of fact, the RSSI is already sent in the debug mode on the 4-wire version.

**Reply by chunkysteveo** [reply]
Cool. I saw the wifi strength in the web console data so assumed it would be possible eventually. I'll keep an eye out for the updates!

**Reply by visualapproach** [reply]
Update's here (6w, development)
